serverless/serverless
# Spotinst - updating docs for configuring spotinst credentials

## What did you implement:

Documentation change for configuring Spotinst credentials



## How did you implement it:

Minor documentation change



## How can we verify it:

check the Spotinst provider docs:
main README
guide/credentials.md
guide/create-token.md



## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
